### ISSUE TITLE
Chaining animations

### ISSUE THREAD

**Issue by philipg-pm** [mindjolt/starling-builder-editor#19]
Hello,

i couldn't find a way for example to have :

T->
[ANIMATION 1] [ ANIMATION 2] 
oooo[ANIMATION 3] [ ANIMATION 4]
oooooooo[ANIMATION 3] [ ANIMATION 4]

And then loop the whole thing ? The animator seems quite poor :(

**Comment by yuhengh** [comment]
Yes, it's based on the Starling Tween class and quite simple. Timeline animation is too much work to implement. You can do more animation by code though, or using the more powerful TweenLite library.
